pajbot/migration_revisions/db/0012_badges.py

```diff
@@ -0,0 +1,3 @@
+def up(cursor, bot):
+    cursor.execute('ALTER TABLE "user" ADD COLUMN IF NOT EXISTS founder BOOLEAN NOT NULL DEFAULT FALSE')
+    cursor.execute('ALTER TABLE "user" ADD COLUMN IF NOT EXISTS vip BOOLEAN NOT NULL DEFAULT FALSE')
```

Comment:
why is there a `IF NOT EXISTS` in there, shouldn't the schema upgrade fail if the column already exists?

Reply:
oh i left it in cause i compressed a bunch of schema changes, ill remove it by bad.